lib/robots/dor_repo/ocr/stage_files.rb

```diff
@@ -9,9 +9,19 @@ def initialize
           super('ocrWF', 'stage-files')
         end
 
-        # available from LyberCore::Robot: druid, bare_druid, workflow_service, object_client, cocina_object, logger
         def perform_work
-          # Copy OCR files from ABBYY output folder to the workspace
+          workspace_dir = object_client.workspace.create
```

Comment:
if my PRs for [DSA](https://github.com/sul-dlss/dor-services-app/pull/5042) and [dor-services-client](https://github.com/sul-dlss/dor-services-client/pull/436) are accepted, once you bump dor-services-client, you can do this: 

`workspace_dir = object_client.workspace.create(content: true)` and then skip creating the content sub-directory below (it will already be there)

Reply:
new dor-services-client gem released, so you can now bump (14.11.0 is what you need) and update that here

deploying updated dsa to QA for testing purposes (we also need the latest version to go with 14.11.0)